packages/e2e-tests/specs/widgets/adding-widgets.test.js

```diff
@@ -0,0 +1,348 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	visitAdminPage,
+	deactivatePlugin,
+	activatePlugin,
+	activateTheme,
+} from '@wordpress/e2e-test-utils';
+
+/**
+ * External dependencies
+ */
+import { groupBy, mapValues } from 'lodash';
+
+/** @typedef {import('puppeteer').ElementHandle} ElementHandle */
+
+describe( 'Widgets screen', () => {
+	beforeEach( async () => {
+		await visitAdminPage( 'themes.php', 'page=gutenberg-widgets' );
+		// Wait for the widget areas to load.
+		await page.waitForSelector(
+			'[aria-label="Block: Widget Area"][role="group"]'
+		);
+	} );
+
+	afterEach( async () => {
+		await cleanupWidgets();
+	} );
+
+	beforeAll( async () => {
+		// TODO: Ideally we can bundle our test theme directly in the repo.
+		await activateTheme( 'twentytwenty' );
```

Comment:
Why specifically TwentyTwenty?

Reply:
It just happens to have 2 widget areas so that we can test DnD and some other stuffs more easily 😅 .